src/core/Akka/Serialization/NewtonSoftJsonSerializer.cs

```diff
@@ -29,6 +29,7 @@ public class NewtonSoftJsonSerializer : Serializer
         private readonly JsonSerializer _serializer;
 
         public JsonSerializerSettings Settings { get { return _settings; } }
+        public object Serializer { get { return _serializer; } }
```

Comment:
To implement this workaround, field must be public (or reacher via reflection) and it must return object and not direct serializer (to maintain Akka.FSharp free from direct dependency on JSON.NET).